test/libc/runtime/mmap_test.c

```diff
@@ -219,6 +221,86 @@ TEST(isheap, mallocOffset) {
   ASSERT_TRUE(_isheap(p + 100000));
 }
 
+static const char *ziposLifePath = "/zip/life.elf";
+TEST(mmap, ziposCannotBeAnonymous) {
```

Comment:
Thank you so much for putting in the additional effort to write tests.

Reply:
It's amazing how much they ease development.